.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+
+name: CI
+on:
+  push:
+    branches:
+      - master
+    tags: '*'
+  pull_request:
+    branches:
+      - master
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix: 
+        version:
+          - '1.6' # LTS
+          - '1' # expands to latest 1.x release
```

Comment:
```suggestion
```

@sglyon would you be fine with only testing on 1.6 for now (all tests pass there!).

I'd appreciate a response here (and #271), QuantEcon is holding back quite a few packages that I'd like to use for assignments.

Reply:
removed this line to ensure that tests will pass